src/bp-templates/bp-nouveau/includes/xprofile/functions.php

```diff
@@ -43,3 +43,55 @@ function bp_nouveau_xprofile_enqueue_scripts() {
 	}
 	wp_enqueue_script( 'bp-nouveau-xprofile' );
 }
+
+
+/**
+ * Return profile header buttons
+ *
+ * @since BuddyBoss 1.5.1
+ *
+ * @return mixed|void
+ */
+
+function bb_profile_header_buttons() {
+
+	$buttons = apply_filters( 'bb_profile_header_buttons', array(
+		'member_friendship' => __( 'Connect', 'buddyboss' ),
+		'member_follow'     => __( 'Follow', 'buddyboss' ),
+		'private_message'   => __( 'Message', 'buddyboss' ),
+		'member_switch'     => __( 'View As', 'buddyboss' ),
+	) );
+
+	return $buttons;
+}
+
+/**
+ * Save Profile Header
+ * @since BuddyBoss 1.5.1
+ */
+
+add_action( 'wp_ajax_save_profile_header_buttons_order', 'save_profile_header_buttons_order' );
+
+function save_profile_header_buttons_order(){
+	$buttons = sanitize_text_field( $_POST['buttons'] );
+
+	/**
+	 * Filter the profile header buttons
+	 *
+	 * @since BuddyBoss 1.5.1
+	 */
+	update_option( '_bb_profile_header_buttons', apply_filters( '_bb_profile_header_buttons', $buttons ) );
+
+	wp_send_json_success();
+}
+
+function bb_get_profile_header_buttons(){
```

Comment:
@shohelbuddyboss this functions should be in bp-templates/bp-nouveau/includes/customizer.php

Again, The function name always starts with bp_

Reply:
If move this function to bp-templates/bp-nouveau/includes/customizer.php the function is not working as expected, it's through undefined function from this file. so moved it to bp-templates/bp-nouveau/includes/functions.php

Added bp_

new function name `bp_get_profile_header_buttons_by_order()`